spec/jobs/catalog_job_spec.rb

```diff
@@ -8,6 +8,10 @@ def update_registry!(catalog)
 end
 
 describe CatalogJob do
+  before :each do
+    VCR.turn_on!
```

Comment:
My gut feeling is that the `handle_push_event` method is to blame :grin: Maybe you should track which test triggers this situation and enable VCR there, or provide a VCR for that case.